components/create/CreateNft.vue

```diff
@@ -144,10 +144,10 @@
       </NeoField>
 
       <!-- royalty -->
-      <NeoField v-if="isBasilisk">
+      <NeoField v-if="isBasilisk || isRemark">
```

Comment:
not available on `ksm` chain (only rmrk2)

Reply:
thanks
fixed